Documentation Edit Request

v-viksur opened the pull request:
In the document, there is a minor correction in the  PowerShell command under the section “Convert standard managed disks to premium”
Mistake command: $vm = Get-AzureRmVM -Name $vmName -rgName $resourceGroupName
Correct command: $vm = Get-AzureRmVM -Name $vmName -ResourceGroupName $rgName

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

v-viksur committed:
Documentation Edit Request

In the document, there is a minor correction in the  PowerShell command under the section “Convert standard managed disks to premium”
Mistake command: $vm = Get-AzureRmVM -Name $vmName -rgName $resourceGroupName
Correct command: $vm = Get-AzureRmVM -Name $vmName -ResourceGroupName $rgName
